src/CoreFoundation/CGAffineTransformComponents.cs

```diff
@@ -0,0 +1,31 @@
+// Copyright 2022 Microsoft Corporation.
+
+#nullable enable
+
+using System;
+using System.Runtime.InteropServices;
+using System.Runtime.Versioning;
+
+using CoreGraphics;
+
+namespace CoreFoundation {
+#if NET
+	[SupportedOSPlatform ("ios")]
+	[SupportedOSPlatform ("maccatalyst")]
+	[SupportedOSPlatform ("macos")]
+	[SupportedOSPlatform ("tvos")]
+#endif
```

Comment:
Is this struct only available in NET?

Reply:
No, but we don't add attributes to structs in legacy usually. We have a test that insists we do in NET6.